qhub_ops/template/{{ cookiecutter.repo_directory }}/.github/workflows/kubessh.yaml

```diff
@@ -0,0 +1,77 @@
+name: "Kubessh Infrastructure"
```

Comment:
This file is identical to the [jupyterhub.yaml](https://github.com/Quansight/qhub-ops/blob/master/qhub_ops/template/%7B%7B%20cookiecutter.repo_directory%20%7D%7D/.github/workflows/jupyterhub.yaml) what is the motivation for adding this file?

Reply:
I agree. How should I make sure that terraform changes to kubessh are applied if a new user is added to config.yaml ? Is this file not necessary ?

Reply:
Whenever a new user is added to the config, the user needs to render the project, which should make changes to the `infrastructure/kubessh.yaml` and since a change is made to `infrastructure` folder, this should trigger a CI run for `jupyterhub.yaml` since `infrastructure` is present in the paths.